src/components/PupilViews/PupilIntro/PupilIntro.view.tsx

```diff
@@ -1,28 +1,189 @@
 import {
+  OakBackLink,
+  OakBox,
+  OakCardHeader,
   OakFlex,
+  OakGrid,
+  OakGridArea,
   OakHeading,
-  OakSecondaryButton,
+  OakIcon,
+  OakLessonBottomNav,
+  OakLessonInfoCard,
+  OakLessonLayout,
+  OakLessonTopNav,
+  OakP,
+  OakPrimaryButton,
+  OakPrimaryInvertedButton,
+  OakSpan,
+  OakStaticMessageCard,
 } from "@oaknational/oak-components";
 
 import { useLessonEngineContext } from "@/components/PupilComponents/LessonEngineProvider";
+import { PupilLessonOverviewData } from "@/node-lib/curriculum-api";
+import { ContentGuidance } from "@/components/TeacherComponents/LessonOverviewRequirements";
 
-export const PupilViewsIntro = () => {
-  const { completeSection } = useLessonEngineContext();
-  return (
-    <OakFlex
-      $flexDirection={"column"}
-      $alignItems={"start"}
-      $pa={"inner-padding-xl"}
-      $gap={"space-between-s"}
-    >
-      <OakHeading tag="h1">Intro</OakHeading>
-      <OakSecondaryButton
+export type PupilViewsIntroProps = PupilLessonOverviewData;
+
+export const PupilViewsIntro = (props: PupilViewsIntroProps) => {
+  const {
+    contentGuidance,
+    supervisionLevel,
+    lessonEquipmentAndResources,
+    worksheetUrl,
+  } = props;
+  const { completeSection, updateCurrentSection } = useLessonEngineContext();
+
+  const topNavSlot = (
+    <OakLessonTopNav
+      backLinkSlot={
+        <OakBackLink
+          type="button"
+          onClick={() => {
+            updateCurrentSection("overview");
+          }}
+        />
+      }
+      heading={"Introduction"}
+      lessonSectionName={"intro"}
+      mobileSummary={
+        <OakSpan $color={"text-primary"} $font={"body-3"}>
+          In progress...
+        </OakSpan>
+      }
+    />
+  );
+
+  const bottomNavSlot = (
+    <OakLessonBottomNav>
+      <OakPrimaryButton
+        type="button"
+        width={["100%", "auto"]}
         onClick={() => {
           completeSection("intro");
         }}
       >
-        Complete Section
-      </OakSecondaryButton>
-    </OakFlex>
+        I'm ready
+      </OakPrimaryButton>
+    </OakLessonBottomNav>
+  );
+
+  const removedGuidanceDuplicates = Array.from(
+    new Set(
+      contentGuidance?.map(
+        (guidance: ContentGuidance) => guidance.contentGuidanceLabel,
+      ),
+    ),
+  );
```

Comment:
Annoying that this is a thing 😭

Reply:
as in the data should already be in a useable format?